src/mon/PaxosService.cc

```diff
@@ -377,6 +377,8 @@ void PaxosService::maybe_trim()
   if (!is_writeable())
     return;
 
+  dout(20) << __func__ << dendl;
```

Comment:
if we want to print out some logging messages, would be better to print more info than just the function name? like
```c++
const version_t trim_to = get_trim_to();
const version_t first_committed = get_first_committed();
dout(20) << __func__ << " " << first_committed << "~" << trim_to << dendl;
```

Reply:
Thanks, done.